event-schemas/schema/core-event-schema/unsigned_prop.yaml

```
@@ -0,0 +1,32 @@
+# Copyright 2020 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+title: UnsignedData
+type: object
+description: Contains optional extra information about the event.
+properties:
+  age:
+    description: The time in milliseconds that has elapsed since the event was
+      sent. This field is generated by the local homeserver, and may be incorrect
+      if the local time on at least one of the two servers is out of sync, which can
+      cause the age to either be negative or greater than it actually is.
+    type: integer
+  redacted_because:
+    description: Optional. The event that redacted this event, if any.
+    title: Event
+    type: object
+  transaction_id:
+    description: The client-supplied transaction ID, if the client being given
```

Maybe something like

```suggestion
    description: The client-supplied transaction ID, for example, provided via ``PUT /_matrix/client/r0/rooms/{roomId}/send/{eventType}/{txnId}``, if the client being given
```

?

---

yea, might as well improve it while we're here. Have applied manually (line wrapping)